File: PE5/PE5.java

public class PE5 {
	public static boolean checksol(int n) {
		for (int i = 1; i <= 20; i++) {
			if (n % i != 0) return false;
		}
		return true;
	}
	
	public static void main (String args[]) {
		int p = 2*3*5*7*11*13*17*19;
		int i = 0;
		while (true) {
			i += p;
			if (checksol(i)) {
				System.out.println(i);
				break;
			}
		}
	}
}

File: PE5/PE5.py
# 2, 3, 5, 7, 11, 13, 17, 19
# Need to be div by all primes under 20
# Need to be div by prod of all primes under 20

def checksol (n):
    for i in range(1, 21):
        if (n % i != 0):
            return False
    return True

p = 2*3*5*7*11*13*17*19
i = 0
while (True):
    i += p
    if (checksol(i)):
        print(i)
        break
